tgui/packages/tgui/styles/interfaces/ClothingBooth.scss

```
@@ -1,31 +1,52 @@
-@use '.././components/Button.scss';
+@use '../base.scss';
+@use '../colors.scss';
 
-.clothingbooth__ {
-  &dropdown {
+.clothingbooth {
+  &__dropdown {
     margin-bottom: 0 !important;
     margin-right: 0 !important;
   }
 
-  &boothitem {
+  &__boothitem {
     padding-left: 0.5rem;
     padding-right: 0.5rem;
-  }
 
-  &boothitem > div {
-    pointer-events: none;
+    &:hover,
+    &--selected {
+      background-color: colors.$good;
+    }
   }
 
-  &boothitem:hover,
-  &boothitem-selected {
-    background-color: Button.$color-selected;
+  &__boothitem > div {
+    pointer-events: none;
   }
 
-  &boothitem:active {
-    background-color: lighten(Button.$color-selected, 30%);
+  &__boothitem:active {
+    background-color: lighten(colors.$good, 30%);
   }
 
-  &preview {
+  &__preview {
     width: 64px;
     height: 128px;
   }
+
+  &__swatch {
+    &_box {
+      margin: 0.2em;
+      &:hover {
+        outline: 0.167rem solid colors.$good;
+      }
+    }
+  }
+
+  &__groupingtag {
+      border-radius: base.$border-radius;
+  }
+}
+
+// If any swatch foreground shapes need custom styling, this is where it lives.
+.clothingbooth__swatch_icon {
+  &_rainbow {
```

```suggestion
  &--rainbow {
```
As we're using BEM here, let's use the "modifier" syntax (as this is a modifier). Requires a change wherever else this is set.